mvn-defaults/pom.xml

```diff
@@ -11,7 +11,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <javadoc.name>nosqlbench</javadoc.name>
 
-        <java.target.version>11</java.target.version>
+        <java.target.version>12</java.target.version>
```

Comment:
We may need the source and release properties as well.
The release property is a newly supported way to specify "source and target", but older javac versions won't recognize it, thus why all three properties were initially provided.

Reply:
The reason to support recognized flags for older javac versions is so that developers get useful error messages when they have an older JDK installed, which is not uncommon.

Reply:
All 3 are already set to the value of this variable: https://github.com/nosqlbench/nosqlbench/blob/master/mvn-defaults/pom.xml#L453 - I can maybe rename it to the `java.version` ? Or you want that we have a flexibility in defining different source/target/release?

Reply:
As long as they are all set, it works. I missed that detail in the review. I think they should be set to the same value in general.